### PR TITLE
components for image, ingredients, all ingredients buttons

### PR DESCRIPTION
**Pull request by DanielekPark**
Created a component and added css styles for ButtonComponent #8.

### CODE REVIEW

File: client/src/App.css [hunk 1]
```diff
@@ -5,7 +5,7 @@
 .roundButton {
   background-color: #182C4A;
   color: #fff;
-  border:1px solid #182C4A;
```
Comment (dunnjacoba) [inline client/src/App.css:8]: Were these spacings added for readability?

**Review by dunnjacoba — APPROVED** [review]
Looks good to merge.

**Comment by dunnjacoba** [comment]
Addresses #8

File: client/src/App.css (hunk 1 above)
Comment (DanielekPark) [reply client/src/App.css:8]: Oh I think my vs code settings does that automatically

**Comment by jraynovic** [comment]
Looks good! I like the neat way of handling the css classes!

**Pull request merged**